itsUk45/teamwork
# #168803198:api edit articl

What does this PR do?

    build a restful api for editing article

Description of Task to be completed

    build the edit feature
    implement json web tokens for authentication
    add test using mocha, chai, chai-http

How should this be manually tested?

    clone this repo
    cd to teamwork folder
    git checkout ft-api-edit-article-168803198
    start testing patch api/v1/articles/<articleId> routes on your test client such as postman

Any background context you want to provide?

    Note that you must create an article at api/v1/articles before patching it 

** What are the relevant pivotal tracker stories?**

     #168803198
